src/main/java/io/github/hypixel_api_wrapper/wrapper/games/bedwars/HypixelBedWars.java

```diff
@@ -1,77 +1,98 @@
 package io.github.hypixel_api_wrapper.wrapper.games.bedwars;
 
-import io.github.hypixel_api_wrapper.wrapper.HypixelLootChestGame;
 import org.json.JSONObject;
 
-public class HypixelBedWars extends AbstractHypixelBedWars implements HypixelLootChestGame {
+public class HypixelBedWars {
 
-    public HypixelBedWars(JSONObject stats) {
-        super(stats);
+    private final String JSONKey;
+    private final JSONObject stats;
+
+    protected HypixelBedWars(JSONObject stats, HypixelBedWarsModes mode) {
+        this.JSONKey = mode.getJSONKey();
```

Comment:
Maybe store the enum instead of just the key here

Reply:
That means each time a method wants to get the key it first needs to go through the enum. I don't see any downsides of taking in the string at the moment.